inc/thorvg.h

```diff
@@ -1341,6 +1341,53 @@ class TVG_API Picture final : public Paint
      * @return The type id of the Picture class.
      */
     static uint32_t identifier() noexcept;
+    
+    
+    /**
+     * @brief Specify segment by marker
+     *
+     * @param[out] marker segment name
+     *
+     * @note Set empty to reset specified segment
+     *
+     * @note Experimental API
+     */
+    Result segment(const std::string& marker);
+    
+    /**
+     * @brief Specify segment by its position
+     *
+     * @param[in] begin  begin position
+     *
+     * @param[in] end  end position
+     *
+     * @note range from 0.0 to 1.0
+     *
+     * @note Experimental API
+     */
+    Result segment(float begin, float end);
+    
+    /**
+     * @brief Return segment duration
+     *
+     * @param[out] dur segment duration
+     *
+     * @note Will call sync()
+     *
+     * @note Experimental API
+     */
+    Result duration(float&dur);
```

Comment:
Using `duration()` and `frameCount()` wouldn't be necessary because the Animation class already provides these APIs. Users could utilize the existing ones instead of introducing new ones. Even these are not proper to Picture class.